pandas/tests/extension/decimal/array.py

```diff
@@ -197,6 +198,25 @@ def _reduce(self, name: str, skipna: bool = True, **kwargs):
             ) from err
         return op(axis=0)
 
+    def _cmp_method(self, other, op):
```

Comment:
this looks generic, can it be *in* ExtensionScalarOpsMixin? (and maybe have that inherit OpsMixin)

Reply:
i tried that and it felt clunky, decided to make it a separate step